app/views/categories/index.html.erb

```diff
@@ -0,0 +1,50 @@
+<nav class="navbar navbar-expand-lg navbar-light" style="background-color: #3778c2;">
+  <a class="navbar-brand text-white" href="#">
+    <img src="<%= asset_path('nav-icon.png') %>" alt="" class="logo" style="margin-left: 10px;">
+  </a>
+  <h1 class="text-center text-white w-100">Categories</h1>
+    <a class="navbar-brand" href="#">
+    <img src="<%= asset_path('search-icon.png') %>" alt="" class="logo" style="margin-left: 10px;">
+  </a>
+</nav>
+
+<% if user_signed_in? %>
+  <div class="nav-link">
+   Welcome <%= link_to "#{current_user.name}", "#" %>
+  </div>
+<% end %>
+
+
+<div class="notifications">
+  <p class="notice"><%= notice %></p>
+  <p class="alert"><%= alert %></p>
+</div>
+
+<p style="color: green"><%= notice %></p>
+
+<div id="categories" class="list-group">
+  <% @categories.each do |category| %>
+    <a href="<%= category_path(category) %>" class="list-group-item list-group-item-action mb-3">
+      <div class="d-flex justify-content-between align-items-center">
+        <div class="d-flex">
+          <div class="mb-4">
+            <%= category.icon.html_safe %>
+          </div>
+          <div>
+            <h5 class="card-title"><%= category.name %></h5>
+          </div>
+        </div>
+        <% total_transactions = category.transactions.sum(:amount) %>
+        <div class="text-lg font-weight-bold">
+          <p>Total Transactions: <%= total_transactions %></p>
+        </div>
+      </div>
+    </a>
+  <% end %>
+</div>
+
+<footer class="navbar navbar-expand-lg navbar-light fixed-bottom" style="background-color: #5fb523;">
+  <div class="container">
+    <%= link_to "Create new category", new_category_path, class: "btn btn-success btn-block", style: "background-color: #5fb523; width: 100%;" %>
```

Comment:
- [ ] Great job so far @RileyManda There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Per the project requirements, you were asked to ensure that you follow these [design guidelines](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), including:
```
- Colors
- Typography: font face, size, and weight.
- Layout: composition and space between elements.
```
1. KIndly ensure that your `navbar` is styled properly and has a size as seen in the given design :+1: 
2. I will suggest you style the `Welcome Nahnah` to look similar to how `Gift credit` was styled right under the `Navbar`.
3. Kindly style your category cards as seen in the Design and ensure that the icons are a little bigger in size as seen in the design. The Text in your design should also have the same colors, font size, font weight, and similar space between the components. Also, you're missing the dollar sign `$` for the amounts :+1: 
4. Lastly, kindly ensure that the outline around the `Create New Category` button is white as seen in the given design :+1: 

| Your Design | The Given Design |
|--------|--------|
| ![image](https://github.com/RileyManda/coincraft/assets/102699958/91041ea3-e416-4b99-87aa-978d9875f234) | ![image](https://github.com/RileyManda/coincraft/assets/102699958/2b78bbc0-96ce-4d3f-a4d6-21660848d505) | 


- [ ] Kindly make similar changes where necessary on the Transactions page as well since both pages follow the same design guidelines :wink: 
- [ ] You can style the `Total Amount` similar to  `Gift Credit `as seen in the image below. Directly under your Details `Navbar` after which the added transactions can be styled as seen below and the `Add Transaction` button replacing `Scan` :wink: 
- Your Design
![image](https://github.com/RileyManda/coincraft/assets/102699958/77716747-3116-475c-a280-0ec6dc9a3ef1)